frontend/src/tests/navbar.test.tsx

```diff
@@ -0,0 +1,27 @@
+import React from "react";
```

Comment:
Could You rename this file as Navbar.test.tsx

Reply:
Also please move this test file to the same location as the component its testing. My reasoning for this is its easier to keep to update tests when the test file is in the same location as the component it is testing. Also makes it easier to find the corresponding test files when we are updating logic within the components